app/view/settings/SeatView.js

```diff
@@ -963,10 +963,22 @@ SDL.SeatView = Em.ContainerView.create({
             content: 'ID'
         }),
 
-        seatModuleUUID: SDL.Label.create({
+        seatModuleUUID: Em.Select.create({
             elementId: 'seatModelCurrent',
             classNames: 'seatModelCurrent',
-            contentBinding: 'SDL.RCModulesController.currentSeatModuleID'
+            change: function() {
+              console.log('CHANGE!')
```

Comment:
@YarikMamykin wtf?

Reply:
@AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/13/commits/0e778e6a9d256cfbf9d86957eea0d3a0286d4448